### PR TITLE
Add delete --attribute option

### PR DESCRIPTION
**Pull request by tommydeboer**
```
mcmd delete --entity-type dataset --attribute name
mcmd delete dataset --attribute name
```

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/system tested
- [x] User documentation updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: mcmd/commands/delete.py [hunk 1]
```diff
@@ -81,6 +92,17 @@ def _delete_entity_type_data(args):
         client.delete(url)
 
 
+def _delete_entity_type_attribute(args):
+    if args.force or (not args.force and io.confirm(
+            'Are you sure you want to delete attribute {} of entity type {}?'.format(args.attribute, args.resource))):
+        io.start('Deleting attribute {} of entity {}'.format(highlight(args.attribute), highlight(args.resource)))
+        response = client.get(urljoin(config.api('rest2'),
```
Comment (dennishendriksen) [inline mcmd/commands/delete.py]: encoding issues (see comments in #118)

File: mcmd/commands/delete.py [hunk 2]
```diff
@@ -81,6 +92,17 @@ def _delete_entity_type_data(args):
...
+                                      'sys_md_Attribute?q=entity=={};name=={}'.format(args.resource,
+                                                                                      args.attribute)))
+        attribute_id = response.json()['items'][0]['id']
+        client.delete(urljoin(config.api('rest2'), 'sys_md_Attribute/{}'.format(attribute_id)))
```
Comment (dennishendriksen) [inline mcmd/commands/delete.py]: encoding issues (see comments in #118)

File: mcmd/commands/delete.py (hunk 1 above)
Comment (tommydeboer) [reply mcmd/commands/delete.py]: Encoding will be done in the `api` module as soon as #118 is merged

File: mcmd/commands/delete.py (hunk 2 above)
Comment (tommydeboer) [reply mcmd/commands/delete.py]: Encoding will be done in the api module as soon as #118 is merged

**Pull request merged**